modules/nw-egress-router-configmap.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/configuring-egress-router-configmap.adoc
+
+[id="configuring-egress-router-configmap_{context}"]
+= Configuring an egress router destination mappings with a ConfigMap
+
+For a large or frequently-changing set of destination mappings, you can use a config map to externally maintain the list.
+An advantage of this approach is that permission to edit the config map can be delegated to users without `cluster-admin` privileges. Because the egress router pod requires a privileged container, it is not possible for users without `cluster-admin` privileges to edit the pod definition directly.
+
+[NOTE]
+====
+The egress router pod does not automatically update when the config map changes.
+You must restart the egress router pod to get updates.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file containing the `EGRESS_DESTINATION` data, as in the following example:
++
```

Comment:
Should this have some sort of source specified? I don't know what it would be, just haven't run into any without a source type since we did that big sweep

Reply:
@jeana-redhat, going to try this instead

> . Create a file containing the mapping data for the egress router pod, as in the following example: